app/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '25.0.0'
+
+    defaultConfig {
+        minSdkVersion 17
+        targetSdkVersion 23
```

Comment:
not necessary. 23 is enough to advertise we are compatible with new permissions setting logic.